LibrarySystemFrontEnd/library-management-system/src/views/AuthorView.vue

```diff
@@ -1,4 +1,6 @@
 <template>
+<div>
```

Comment:
Try to remove newly added div (only add navbar tag) and see whether it is working - Try it for all views except the home page